numerics/src/LCP/LinearComplementarityProblem_as_ConvexQP.c

```diff
@@ -18,7 +18,7 @@
 */
 #include <stdlib.h>
 #include <assert.h>
-#include <math.h>
+#include <tgmath.h>
 #include "LinearComplementarityProblem_as_ConvexQP.h"
```

Comment:
Perhaps not.  Previously these files used `max` not `fmax` so I was trying to maintain semantics since `max` doesn't compile for me with BUILD_AS_CPP.

> /home/sinclairs/projects/siconos/numerics/src/LCP/lcp_compute_error.c:52:45: error: no matching function for call to ‘max(int, double)’

However yes I can confirm that changing `max` to `fmax` and not changing `math.h` to `tgmath.h` also works.

Reply:
Looks good